### PR TITLE
accounts/abi: added output parsing & added call mechanism

### PR DESCRIPTION
**Pull request by obscuren**
Added calling mechanism and return value parsing

### CODE REVIEW

**Comment by robotally** [comment]
| Vote | Count | Reviewers |
| :---: | :---: | :---: |
| :+1: | 0 |  |
| :-1: | 0 |  |

_Updated: Wed Nov 25 11:25:10 UTC 2015_

File: accounts/abi/type.go [hunk 1]
```diff
@@ -31,6 +31,7 @@ const (
 	BoolTy
 	SliceTy
 	AddressTy
+	HashTy
```
Comment (zelig) [inline accounts/abi/type.go]: cool you added the hash type. was gonna suggest :)

**Comment by fjl** [comment]
:+1:

**Comment by bas-vk** [comment]
:+1:

**Pull request merged**